### PR TITLE
Pdf

### PR DESCRIPTION
**Pull request by gduck**
I used html2canvas, which puts a good representation of my dom onto a canvas. Then I convert the canvas to .png and open it in print. Seems to work pretty well, except it opens a new window tab for some reason

### CODE REVIEW

**Comment by h8rry** [comment]
Let's try not to include the whole javascript library in your version control (git). Try to find a rails gem that wraps around this library. You can bower it here: https://github.com/niklasvh/html2canvas

**Comment by h8rry** [comment]
Are you using `html2canvas` instead of the default browser print function?

File: app/assets/stylesheets/print_old.css.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*#newspaper {
```
Comment (h8rry) [inline app/assets/stylesheets/print_old.css.scss]: Can we delete this file now?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -3,9 +3,12 @@
 <head>
   <title>Newz</title>
   <%= stylesheet_link_tag 'application', media: 'all' %>
+  <link rel="stylesheet" type="text/css" media="print" href="assets/print.css.scss">
   <%= javascript_include_tag 'application' %>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.9.0/moment.min.js"></script>
   <script type="text/javascript" src="https://cdn.jsdelivr.net/angular.moment/0.9.0/angular-moment.min.js"></script>
+  <script type="text/javascript" src="assets/html2canvas/html2canvas.js"></script>
```
Comment (h8rry) [inline app/views/layouts/application.html.erb]: You should do this in `application.js`

File: app/assets/javascripts/angular/controller/home-controller.js.erb [hunk 3]
```diff
@@ -87,14 +87,65 @@ app.controller('HomeCtrl', ['$scope', '$http', '$rootScope', '$timeout', functio
...
+          // console.log(tWindow); 
+          $(tWindow.document.body).html("<img id='Image' src=" + myImage + " style='width:100%;'></img>").ready(function () {
+              tWindow.focus();
+              tWindow.print();
+          });
+        }
+      });
+      // VARIOUS OPTIONS I'VE TRIED
+      // I WOULD LIKE TO KEEP THEM FOR REFERENCE
+      // FOR NOW
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/home-controller.js.erb:111]: Thanks

File: app/assets/stylesheets/print_old.css.scss (hunk 1 above)
Comment (gduck) [inline app/assets/stylesheets/print_old.css.scss]: yes. and the new one too - no longer needed. it's gone

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (gduck) [inline app/views/layouts/application.html.erb]: yes. that will fix heroku too. i'll push to heroku now

File: app/assets/stylesheets/newspaper.css.scss [hunk 4]
```diff
@@ -13,6 +13,9 @@
 
 
 /* MAIN LAYOUT ----------*/
+#header, .site-container {
+  display: "";
+}
```
Comment (h8rry) [inline app/assets/stylesheets/newspaper.css.scss]: What are we doing here?

Comment (gduck) [inline app/assets/stylesheets/newspaper.css.scss]: removed - in fillers branch

**Pull request merged**